runtime/src/main/java/io/quarkiverse/messaginghub/pooled/jms/PooledJmsDecorator.java

```diff
@@ -12,6 +12,8 @@
 import jakarta.jms.JMSContext;
 import jakarta.jms.JMSException;
 
+import io.quarkus.arc.impl.Reflections;
```

Comment:
You should not use `io.quarkus.arc.impl.Reflections` in your code. It's an implementation detail, i.e. not considered a public API and should only be used internally. It's also mentioned in the javadoc.

Reply:
Yeah, I understand.